include/tvm/relay/attrs/nn.h

```diff
@@ -120,7 +120,7 @@ struct Conv2DAttrs : public tvm::AttrsNode<Conv2DAttrs> {
   tvm::String data_layout;
   tvm::String kernel_layout;
   tvm::String out_layout;
-  std::string auto_scheduler_rewritten_layout;
+  std::string auto_scheduler_rewritten_layout;  // The layout after auto-scheduler's layout rewrite
```

Comment:
```suggestion
  /*! \brief The layout after auto-scheduler's layout rewrite. */
  tvm::String auto_scheduler_rewritten_layout;
```

Other members all used `tvm::String`.